combat.md

```diff
@@ -1,3 +1,51 @@
 # Combat
 
+Combat in Blood Down the Gullet should feel mostly familiar to veterans of 4E. You'll roll initiative to determine the turn order for this combat. Within each turn, you'll have one move, one minor, and one standard action to take, with the ability to trade any down to a lesser action (taking an additional move instead of a standard, for example). You'll have a small library of ability cards as part of your character sheet, most of which will involve rolling a d20, adding some base score, and comparing the result to an enemy's defense value.
+
+The major differences that you'll notice will be in how skills are consumed and recharged, and how movement is tracked.
+
+## Skill Charges
+
+You still have **At-Will** abilities, which you can use as many times as you like. However, you'll no longer have encounter and daily powers. Instead, each non-At-Will ability will have a *charge number* on it.
+
+At the beginning of the day, put a number of charge tokens on each ability card corresponding to its charge number. Each time that you use one of those abilities, remove a charge token. Once the charge tokens are gone, you can't use the ability any more that day.
+
+Some abilities may give you the choice of using additional charge tokens to enhance their effect somehow. You have to choose to spend the tokens before you roll.
+
+Some abilities may allow you to restore charge tokens to other characters or yourself. The granted tokens can be placed on any ability with a charge number. You *may* add additional tokens above the charge number. They'll persist until you use them, or the next extended rest, whichever comes first.
+
+After a combat encounter, you'll naturally regain five charge tokens in the same manner. After an extended rest, all abilities reset to the number of charge tokens equal to their charge number.
```

Comment:
"Five" is a wild guess, here. I'll have to see how a few encounters go and adjust accordingly.